doc/releases/changelog-dev.md

```diff
@@ -291,6 +291,9 @@ the `decimals` and `show_matrices` keywords are added. `qml.drawer.tape_text(tap
 
 <h3>Bug fixes</h3>
 
+* Call `pip show` with the subprocess library to avoid outputting a common warning.
+  [(#2422)](https://github.com/PennyLaneAI/pennylane/pull/2422)
+
```

Comment:
@timmysilv feel free to add yourself to the list of contributors below :+1:

Reply:
done!